src/classes/OPP_AutomatedSoftCreditsService.cls

```diff
@@ -38,6 +38,10 @@
 public class OPP_AutomatedSoftCreditsService {
 
     private static final String RELATED_OPPORTUNITY_CONTACT_ROLE_NONE = '--None--';
+    private static final String OBJECT_TYPE_RELATIONSHIP = 'npe4__Relationship__c';
+    private static final String OBJECT_TYPE_AFFILIATION = 'npe5__Affiliation__c';
+    private static final String FILED_NAME_RELATED_CONTACT = 'npe4__RelatedContact__c';
+    private static final String FILED_NAME_CONTACT = 'npe5__Contact__c';
```

Comment:
To be consistent with the above constant, should this one be `FIELD_NAME_AFFILIATED_CONTACT`?

Reply:
It is consistent based on the field names. The field for Relationships is RelatedContact__c whereas the field name for Affiliations is Contact__c. However, for readability, I can change the name.